src/cs/Ssh/SshChannel.cs

```diff
@@ -428,7 +428,7 @@ private async Task SendEofAsync(CancellationToken cancellation)
 					Trace.TraceEvent(
 						TraceEventType.Error,
 						SshTraceEventIds.ChannelRequestFailed,
-						$"Channel request failed with exception ${ex.ToString()}.");
+						$"Channel request failed with exception {ex.ToString()}.");
```

Comment:
Nit: just `{ex}`. It calls `.ToString()` on its own. Also if there is any format provider in `Trace`, it will handle exception object better than a string.